packages/main/src/components/AnalyticalTable/index.tsx

```diff
@@ -539,7 +540,7 @@ const AnalyticalTable = forwardRef((props: AnalyticalTablePropTypes, ref: Ref<HT
     return props.data;
   }, [props.data, minRows]);
 
-  const tableInstanceRef = useConsolidatedRef<Record<string, any>>(tableInstance);
+  let tableInstanceRef = useRef<Record<string, any>>();
```

Comment:
```suggestion
  const tableInstanceRef = useRef<Record<string, any>>(null);
```
`tableInstanceRef` is never reassigned, so I would prefer const. In addition, using null as default value makes it a `MutableRefObject` :)